scripts/aruco_NordSud.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+# -*-coding:Latin-1 -*
+import rospy
+
+from std_msgs.msg import Header, Time, Bool
+from fiducial_msgs.msg import FiducialArray, Fiducial
+
+#class Aruco_properties:
+#def __init__(self):
+ #   self.start = False
+    
+    #self.pub_fiducial_array_topic = rospy.Publisher(self.dummy_fiducial,FiducialArray, queue_size=1)
+
+
+def getFiducialArray(self, data):
+    fiducial_array = FiducialArray()
```

Comment:
`data` ici est de type FiducialArray, c'est lui qui contient les valeurs que tu cherches

```suggestion
def getFiducialArray(self, data):
    fiducial_array = data
```